data/step_function_inputs/NASA_JPL_global_agb_mean_2020.json

```diff
@@ -3,5 +3,7 @@
     "discovery": "inventory",
     "inventory_url": "s3://maap-user-shared-data/nasa_jpl_global_agb_mean_2020/inventory.csv",
     "cogify": false,
-    "upload": false
+    "upload": false,
+    "metadata_file_url_key": "s3_path",
```

Comment:
Oh my bad, I immediately assumed it was this since the pipeline was broken but used to work. 

Is it another parameter I was supposed to use ? Need to look at it tomorrow.

Reply:
Hm I'm missing something, the pipeline should've worked without these changes...